addons/addons.json

```diff
@@ -116,7 +116,7 @@
   "search-sprites",
   "hide-signatures",
   "disable-sprite-wobble",
-
+  "floating-navbar"
```

Comment:
uhh this should have a comma after it 
also add the line break back

Reply:
Fixed

Reply:
[Resolved](https://github.com/ScratchAddons/ScratchAddons/pull/4155/commits/f323eed81d55f060fd3e7cccb87e5b0fd95f3260)